app/middlewares/passArg.js

```diff
@@ -0,0 +1,4 @@
+exports.passArgToUserLocals = value => (_, res, next) => {
+  res.locals.user = value;
+  next();
+};
```

Comment:
If you add a default value in the model and separate the process for administrator users this would not be necessary.